modules/issue_tracker/jsx/IssueForm.js

```diff
@@ -243,7 +243,10 @@ class IssueForm extends Component {
             label='Assignee'
             emptyOption={true}
             options={this.state.Data.assignees}
-            disabledOptions={this.state.Data.inactiveUsers}
+            disabledOptions={this.state.Data.inactiveUsers.reduce(
+              (disabledOpts, inactiveUser) =>
+                (disabledOpts[inactiveUser] = '', disabledOpts), {}
+            )}
```

Comment:
@laemtl I am having this error in js `Uncaught TypeError: this.state.Data.inactiveUsers.reduce is not a function`.